wallets/rn_cli_wallet/src/utils/theme.ts

```diff
@@ -0,0 +1,30 @@
+import {useColorScheme} from 'react-native';
+
+const theme = {
```

Comment:
there's already a ThemeUtil file, is this one needed?

Reply:
Moved this to `rn_wallet_cli` while copy the changes from `Web3InboxLab` app, not necessary now, removing